### PR TITLE
Add compile time errors on interop calls of constructors of abstract classes as breaking changes

### PR DESCRIPTION
**Pull request by gabilang**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -1273,6 +1273,16 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
 
 - To avoid clashes with Java identifiers, the character used for encoding and decoding identifiers has been changed from `$` to `&`.
 
+- Interop calls to constructors of Java abstract classes has been restricted at compile time.
```
Comment (HindujaB) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]:
``` suggestion
- Interop calls to Java abstract class constructors have been restricted at compile time.
```

**Comment by sm1990** [comment]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -1273,6 +1273,16 @@ To view bug fixes, see the GitHub milestone for Swan Lake Update 9 (2201.9.0) of
 
 - To avoid clashes with Java identifiers, the character used for encoding and decoding identifiers has been changed from `$` to `&`.
 
+- A bug which resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.
```
Comment (HindujaB) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]:
```suggestion
- A bug that resulted in no compilation errors for interop calls of Java abstract class constructors has been fixed.
```

Comment (warunalakshitha) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]: Should we mention interfaces as well

Comment (ravinperera00) [inline downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]:
```suggestion
- A bug that allowed interop calls of Java abstract class constructors to proceed without generating compilation errors has been fixed.
```
This sounds a bit more explanatory. Is it too long?

Comment (gabilang) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]: Yes, it's applicable for interfaces too.

Comment (gabilang) [reply downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md:1276]: Stick with the original sentence to follow the writing style of this doc.

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Review by sm1990 — APPROVED** [review]

**Pull request merged**